cmd/bootkube/render.go

```diff
@@ -100,7 +100,8 @@ func validateRenderOpts(cmd *cobra.Command, args []string) error {
 		return errors.New("You must specify either all or none of --etcd-ca-path, --etcd-certificate-path, and --etcd-private-key-path")
 	}
 	if renderOpts.etcdCertificatePath != "" && renderOpts.selfHostedEtcd {
-		return errors.New("Cannot specify --etcd-certificate-path with --experimental-self-hosted-etcd")
+		return errors.New("Cannot specify --etcd-certificate-path with --experimental-self-hosted-etcd." +
+			" Self-hosted etcd + TLS will auto-generate certs based on root CA cert.")
```

Comment:
Yeah. This part is definitely something I'm gonna fix in next PR.
I want this PR to be more about autogen self-hosted certs.
I already have the entire work done. I'm separating them into smaller PRs so that it eases review work and make it good enough to pass out tests and multi-node working.
Let's take this in the following PR.